test/axelar-gateway.js

```diff
@@ -0,0 +1,184 @@
+const { SuiClient, getFullnodeUrl } = require('@mysten/sui.js/client');
+const { Ed25519Keypair } = require('@mysten/sui.js/keypairs/ed25519');
```

Comment:
see lint description for info and workarounds/disable it
https://typescript-eslint.io/rules/no-var-requires/